pkg/provisioner/env.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright (c) 2023 ApeCloud, Inc. All rights reserved.
```

Comment:
whether we need to retain the original copyright from openebs, then add PORT copyright

Reply:
```
//
// Portions Copyright (c) 2023 ApeCloud, Inc.
//
// Copyright 2006, Google Inc.
// All rights reserved.
```

Reply:
Updated.